src/Layout/QueryParamContext.tsx

```diff
@@ -337,18 +341,18 @@ export default function QueryParamProvider({ children }: any) {
     }
   }, [router.isReady]);
 
-  React.useEffect(() => {
-    if (router.query.tenant) {
-      // TODO => verify tenant before setting it
-      settenant(router.query.tenant);
-      localStorage.setItem("tenant", router.query.tenant);
-    } else {
-      localStorage.removeItem("tenant");
-    }
-    return () => {
-      localStorage.removeItem("tenant");
-    };
-  }, [router.query.tenant]);
+  // React.useEffect(() => {
+  //   if (router.query.tenant) {
+  //     // TODO => verify tenant before setting it
+  //     settenant(router.query.tenant);
+  //     localStorage.setItem("tenant", router.query.tenant);
+  //   } else {
+  //     localStorage.removeItem("tenant");
+  //   }
+  //   return () => {
+  //     localStorage.removeItem("tenant");
+  //   };
+  // }, [router.query.tenant]);
```

Comment:
If we do not need this anymore, let's remove this.

Reply:
done